scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -0,0 +1,294 @@
+			basename = basename,
+			mode = "snp",
+			model_backend = model_backend,
+			python_script = training_python_script,
+			hyperparameters_json = hyperparameters_json,
+			gatk_override = gatk_override,
+			gatk_docker = gatk_docker
+	}
+
+	call TrainVariantAnnotationModel as TrainVariantAnnotationModelINDELs {
+		input:
+			annots = ExtractVariantAnnotationsINDELs.annots,
+			basename = basename,
+			mode = "indel",
+			model_backend = model_backend,
+			python_script = training_python_script,
+			hyperparameters_json = hyperparameters_json,
+			gatk_override = gatk_override,
+			gatk_docker = gatk_docker
+	}
+
+	scatter(idx in range(length(vcf))) {
+		call ScoreVariantAnnotations as ScoreVariantAnnotationsSNPs {
+			input:
+				vcf = vcf[idx],
+				vcf_index = vcf_index[idx],
+				basename = basename,
+				mode = "SNP",
+				model_backend = model_backend,
+				python_script = scoring_python_script,
+				annotations = snp_annotations,
+				extracted_training_vcf = ExtractVariantAnnotationsSNPs.extracted_training_vcf,
+				extracted_training_vcf_index = ExtractVariantAnnotationsSNPs.extracted_training_vcf_index,
+				interval_list = score_interval_list,
+				model_files = TrainVariantAnnotationModelSNPs.outputs,
+				resource_args = snp_resource_args,
+				use_allele_specific_annotations = use_allele_specific_annotations,
+				gatk_override = gatk_override,
+				gatk_docker = gatk_docker
+		}
+
+		call ScoreVariantAnnotations as ScoreVariantAnnotationsINDELs {
+			input:
+				vcf = vcf[idx],
+				vcf_index = vcf_index[idx],
+				basename = basename,
+				mode = "INDEL",
+				model_backend = model_backend,
+				python_script = scoring_python_script,
+				annotations = indel_annotations,
+				extracted_training_vcf = ExtractVariantAnnotationsINDELs.extracted_training_vcf,
+				extracted_training_vcf_index = ExtractVariantAnnotationsINDELs.extracted_training_vcf_index,
+				interval_list = score_interval_list,
+				model_files = TrainVariantAnnotationModelINDELs.outputs,
+				resource_args = indel_resource_args,
+				use_allele_specific_annotations = use_allele_specific_annotations,
+				gatk_override = gatk_override,
+				gatk_docker = gatk_docker
+		}
+
+	}
+
+	output {
+		Array[File] indels_variant_scored_vcf = ScoreVariantAnnotationsINDELs.output_vcf
+		Array[File] indels_variant_scored_vcf_index = ScoreVariantAnnotationsINDELs.output_vcf_index
+		Array[File] snps_variant_scored_vcf = ScoreVariantAnnotationsSNPs.output_vcf
+		Array[File] snps_variant_scored_vcf_index = ScoreVariantAnnotationsSNPs.output_vcf_index
+	}
+
+}
+
+task ExtractVariantAnnotations {
+	input {
+		String gatk_docker
+		File? gatk_override
+		File input_vcf
+		File input_vcf_index
+		String basename
+		String mode
+		String annotations
+		String resource_args
+		File? interval_list
+		Boolean use_allele_specific_annotations
+
+		Int memory_mb = 14000
+		Int command_mem = memory_mb - 1000
+	}
+	Int disk_size = ceil(size(input_vcf, "GB") + 50)
+	command {
+		set -e
```

Comment:
I agree that it would be nice, in general.  And certainly something we should do in our code.  But this is another piece of Sam's code, and one that we have explicitly tagged for updating soon with his latest version in master.  It doesn't seem to make sense to go in and edit it in this way at this point.